### DISCUSSION TITLE
Actions not being triggered by Midi after learn input assigned

### DISCUSSION THREAD

**Discussion by lenquist0** [CommandPost/CommandPost#3254] [Technical Support] [answered]
Hello!
When I click the learn button in the Midi panel of Control surface settings I'm getting this error in the debug console:

> ERROR:   LuaSkin: hs.midi:callback error:No MIDI Device End Points detected.

Even though it will still detect the midi input presses on my midi controller after pressing "learn", it will not trigger any assigned actions once they are learned.

Help would be appreciated, thank you!

![Screen Shot 2023-08-14 at 8 06 42 PM](https://github.com/CommandPost/CommandPost/assets/142284165/bb46c891-c1f6-4956-9d28-489a5d9fdecb)

**Comment by lenquist0** [comment, marked as answer]
Okay I ended up solving this issue by changing the Bank setting to 2. Not sure why this fixed it, but now it works!

**Comment by lxne** [comment]
Had the same issue. Changing to 2 did solve it.
Back to 1 and it works, too.
